src/AppBundle/Controller/InterviewController.php

```diff
@@ -306,4 +311,145 @@ public function bulkAssignAction(Request $request)
                 )),
         ));
     }
+
+    /**
+     * @param string $responseCode
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    private function userCancelFinal(string $responseCode, Interview $interview)
+    {
+        if ($interview === null) {
+            throw $this->createNotFoundException();
+        }
+
+        $interview->cancel();
+        $manager = $this->getDoctrine()->getManager();
+        $manager->persist($interview);
+        $manager->flush();
+
+        $this->notifyInterviewer(InterviewStatusType::CANCELLED, $interview);
```

Comment:
Her kan du like greit utføre direkte:
`$this->get('app.interview.manager')->sendCancelEmail($interview);`